html_app/watch.py

```diff
@@ -1,125 +1,248 @@
-# <script>$(function(){ var jquery = $.noConflict();if(! window.$ ) { window.$ = jquery; } starcellbio('#main',{});}); window.clearCookie = function() { document.cookie='sessionid=\"invalid\"'};</script>
+js = dict()
+css = dict()
+TIME = '?_=' + str(int(time.time()))
+
+CSS_PREFIX = '<link type="text/css" href="/static/'
+CSS_SUFFIX = '" rel="Stylesheet" />\n'
+
+JS_PREFIX = '<script type="text/javascript" src="/static/'
+JS_SUFFIX = '" charset="UTF-8"></script>\n'
+
+HTML_PREFIX = """
+<!DOCTYPE html>
+<html>
+  <head>
+    <META http-equiv='Content-Type' content='text/html; charset=UTF-8'>
+    <meta http-equiv='content-type' content='text/html; charset=utf-8'>
+    <title>StarCellBio Prototype</title>
+"""
+HTML_SUFFIX = """
+</head>
+<body>
+  <div id='main'></div>
+  <script>
+    $(function() {
+      starcellbio('#main', {});
+    });
+    window.clearCookie = function() {
+      document.cookie = 'sessionid="invalid"'
+    };
+  </script>
+</body>
+"""
 # add raven
-html_suffix += "<script src='//cdn.ravenjs.com/1.1.11/jquery,native/raven.min.js'></script> <script> Raven.config('https://7845856fc975496e8dc2130b7140b19c@app.getsentry.com/19461', { whitelistUrls: ['starcellbio.mit.edu'] }).install(); </script>"
+HTML_SUFFIX += """
+<script src='//cdn.ravenjs.com/1.1.11/jquery,native/raven.min.js'></script>
+<script>
+  Raven.config(
+    'https://7845856fc975496e8dc2130b7140b19c@app.getsentry.com/19461',
+    { whitelistUrls: ['starcellbio.mit.edu'] }
+  ).install();
+</script>
+"""
+
 
 def index_html():
-    global css,js
-    if( js.has_key( 'js/jquery-1.7.2.min.js' )):
-        js.pop('js/jquery-1.7.2.min.js')
-    if( js.has_key( 'js/tinymce.min.js' )):
-        js.pop('js/tinymce.min.js')
-    if( js.has_key( 'js/jquery.tinymce.min.js' )):
-        js.pop('js/jquery.tinymce.min.js')
-    if( js.has_key( 'starcellbio.app.js' )):
-        js.pop('starcellbio.app.js')
-    if( js.has_key( 'swipe/Gruntfile.js' )):
-        js.pop('swipe/Gruntfile.js')
-    if( js.has_key( 'js/soyutils.js' )):
-        js.pop('js/soyutils.js')
-    if( js.has_key( 'master_model.js' )):
-    	js.pop('master_model.js')
-
-    css_join = css_prefix + (time+css_suffix+css_prefix).join(css.keys())+ time + css_suffix
-    js_join = js_prefix + (time+js_suffix+js_prefix).join(js.keys())  + js_suffix
-    js_join = js_prefix + "../scb/get_user.js" + js_suffix + js_join
-    js_join = js_prefix + "starcellbio.app.js" + js_suffix + js_join
-    js_join = js_prefix + "js/soyutils.js" + js_suffix + js_join
-    js_join = js_prefix + "js/tinymce.min.js" + js_suffix + js_join
-    js_join = js_prefix + "js/jquery.tinymce.min.js" + js_suffix + js_join
-    js_join = js_prefix + "js/jquery-1.7.2.min.js" + js_suffix + js_join
-    js_join = js_join + js_prefix + "master_model.js" + js_suffix
-
-
-    return html_prefix + css_join + js_join + html_suffix
+    global css, js
```

Comment:
I know you aren't changing stuff here, so it won't affect the review, but what the what